contrib/storage-hbase/src/main/java/org/apache/drill/exec/store/hbase/HBaseRecordReader.java

```diff
@@ -124,6 +124,11 @@ public HBaseRecordReader(Connection connection, HBaseSubScan.HBaseSubScanSpec su
     } else {
       rowKeyOnly = false;
       transformed.add(ROW_KEY_PATH);
+      /* DRILL-5137 - optimize count(*) queries on MapR-DB Binary tables */
+      if (isSkipQuery()) {
```

Comment:
I thought of this, however I got stuck because there is no vectorContainerWriter in HBaseRecordReader class. If you can send a diff of how to do this, I can include that too. Thanks!